Gemfile

```diff
@@ -1,19 +1,16 @@
-# frozen_string_literal: true
```

Comment:
And what about nodejs. Is there a way to pin its version here ? Or that's something RPM-specific ?